docs/docsite/rst/iocage_inventory_guide_tags.rst

```diff
@@ -0,0 +1,120 @@
+..
+  Copyright (c) Ansible Project
+  GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+  SPDX-License-Identifier: GPL-3.0-or-later
+
+.. _ansible_collections.community.general.docsite.guide_iocage.iocage_inventory_guide.iocage_inventory_guide_tags:
+
+Tags
+----
+
+Quoting `man iocage <https://man.freebsd.org/cgi/man.cgi?query=iocage>`_:
+
+   PROPERTIES
+   ...
+   notes="any string"
+   Custom notes for miscellaneous tagging.
+   Default: none
+   Source: local
```

Comment:
Shouldn't this be a code block (with language `text`)?

Reply:
In that example you were quoting text (with regular text formatting); here it looks more like some preformatted text that should be printed verbatim. I don't know how RST quotes behave with multi-line text, whether you get line-breaks at the exact positions or whether it will end up in one line.

It might be better to have empty lines between the lines if you want to keep the quoting:
```suggestion
   PROPERTIES

   ...

   notes="any string"

   Custom notes for miscellaneous tagging.

   Default: none

   Source: local
```

Also note that in the `man` page, you have indents. A code block (with reduced indents) is probably the better choice to quote a man page's content.

Reply:
I'll add the code block.